src/main/java/com/mathworks/ci/MatlabBuilderConstants.java

```diff
@@ -34,4 +33,10 @@
     
     //Test runner file prefix 
     static final String MATLAB_TEST_RUNNER_FILE_PREFIX = "test_runner_";
+    
+    // MATLAB runner script
+    static final String TEST_RUNNER_SCRIPT = "testScript = genscript(${PARAMS});\n" + "\n"
+            + "disp('Running MATLAB script with content:\\n');\n"
```

Comment:
I don't think `disp` recognizes special characters like `\n`. `disp` will automatically add a new line though, so you probably don't need `\n` here.

Reply:
Done.